src/lib/crypto/kimchi_bindings/stubs/src/linearization.rs

```diff
@@ -1,13 +1,40 @@
 use kimchi::{
-    circuits::expr::Linearization,
+    circuits::{
+        constraints::FeatureFlags,
+        expr::Linearization,
+        lookup::lookups::{LookupFeatures, LookupPatterns},
+    },
     linearization::{constraints_expr, linearization_columns},
 };
 
 /// Converts the linearization of the kimchi circuit polynomial into a printable string.
 pub fn linearization_strings<F: ark_ff::PrimeField + ark_ff::SquareRootField>(
+    uses_custom_gates: bool,
 ) -> (String, Vec<(String, String)>) {
-    let evaluated_cols = linearization_columns::<F>(None);
-    let (linearization, _powers_of_alpha) = constraints_expr::<F>(None, true);
+    let features = if uses_custom_gates {
```

Comment:
I am not sure I follow the logics of this naming. Meaning, if the circuit uses custom gates, then it means that it must have some special features. To me it looks like `uses_custom_gates = false` would have been assigned to `features = None`.

Reply:
Look at the definition of the function. `None` is 'with flags',`Some(flags)` is without flags, with features